src/main/java/org/syncninja/service/DirectoryService.java

```diff
@@ -0,0 +1,31 @@
+package org.syncninja.service;
+
+import org.syncninja.model.Branch;
+import org.syncninja.model.Directory;
+import org.syncninja.repository.DirectoryRepository;
+import org.syncninja.util.ResourceBundleEnum;
+
+public class DirectoryService {
+    DirectoryRepository directoryRepository = new DirectoryRepository();
```

Comment:
Can we make the initialization in the default constructer, it is a better practice to do that.